builder-base/install.sh

```diff
@@ -70,6 +84,27 @@ sha256sum -c $BASE_DIR/buildkit-checksum
 tar -C /usr -xzf buildkit-$BUILDKIT_VERSION.linux-amd64.tar.gz
 rm -rf buildkit-$BUILDKIT_VERSION.linux-amd64.tar.gz
 
+if [[ "$CI" == "true" ]]; then
```

Comment:
When the script is run from inside a Dockerfile, this is false because we don't pass in that environment variable to the build right?

Reply:
Correct, this is the same conditional the script is using above to set the BASEDIR var only in the case thats its not running in the docker container.